controllers/login.js

```diff
@@ -20,11 +21,32 @@ router.post('/login', (req, res) => {
         res.status(200);
         return res.json(data[0]);
     })
-    .catch(err => { res.sendStatus(500) });
+    .catch(err => { 
+        res.sendStatus(500) 
```

Comment:
`logger.error(err)` should be before the sendStatus since the sendStatus might cause this function to end.  Also, can you use `res.status(500)` and then send back a json response with errMessage like you did on line 19?